satellite/audit/verifier.go

```diff
@@ -357,8 +375,13 @@ func (verifier *Verifier) Reverify(ctx context.Context, path storj.Path) (report
 				}
 			}
 			if !found {
+				// node is no longer in pointer, so remove from containment
```

Comment:
The question is that when the segment gets deleted for a node, should we 
1. remove it from containment,
2. pick another piece/segment, or
3. it doesn't matter.

Reply:
I think picking another piece/segment might be the ideal solution, but that is probably too complex of a change to do for now. If it is a bad node, it should end up in containment again anyway even if we remove the node from containment here.